test/nn/test_multihead_attention.py

```diff
@@ -589,7 +589,8 @@ def test_multihead_self_attn_two_masks_fast_path_mock(self, device):
         Multihead self-attention should take fast path when both attention mask (mask type 0)
         and key padding mask (mask type 1) are provided at the same time on CPU and CUDA
         """
-        if device not in ['cpu', 'cuda']:
+        device = device.rstrip(':0123456789')
```

Comment:
why this change?

Reply:
> why this change?

When running on a device such as cuda, the device will have a suffix card number(such as 'cuda:0'), thereby skipping the test case

Reply:
ah, so the test was being erroneously skipped in CI before :o good catch